builtin/credential/radius/path_login.go

```diff
@@ -144,6 +144,9 @@ func (b *backend) RadiusLogin(ctx context.Context, req *logical.Request, usernam
 	packet := radius.New(radius.CodeAccessRequest, []byte(cfg.Secret))
 	UserName_SetString(packet, username)
 	UserPassword_SetString(packet, password)
+	if cfg.NasIdentifier != "" {
+		NASIdentifier_AddString(packet, cfg.NasIdentifier)
```

Comment:
Given that some of the other variables are choosing `Set` functions, is there a reason for choosing `NASIdentifier_AddString` over `NASIdentifier_SetString`? Will `add`ing allocate the space needed?

Reply:
Using Add allows multiple. I don't think it matters here though since we're only supporting a single identifier (at least for now). But if we ever supported more than one we'd need to iterate over and use Add, so makes sense to keep it in for now I think.